tasks/amplify.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Check if Amplify Agent is installed
+  package:
+    name: nginx-amplify-agent
+    state: present
+  ignore_errors: true
+  register: amplify_agent_installed
+
+- name: Install Amplify Agent if not installed
+  block:
+    - name: Download Amplify Agent script
+      get_url:
+        url: "{{ nginx_amplify_script_url }}"
+        dest: "{{ nginx_amplify_script_path }}"
+
+    - name: Run Amplify Agent install.sh script
+      command: "sh /tmp/install-amplify-agent.sh -y"
+      environment:
+        API_KEY: "{{ amplify.api_key }}"
+      become: true
```

Comment:
The reason why we need root here is because the SH script uses `yum` to download the Nginx Amplify packages and any dependencies. Additionally the `-y` flag instructs the script to assume user approval when install packages, much like when installing packages with `yum`.